google/cloud/sqlalchemy_spanner/requirements.py

```diff
@@ -68,6 +68,11 @@ def isolation_level(self):
     def sequences(self):
         return exclusions.closed()
 
+    @property
+    def temporary_tables(self):
+        """target database supports temporary tables"""
```

Comment:
```suggestion
        """Target database supports temporary tables."""
```

Reply:
Oh, yeah. I've copy pasted the comment from the original SQLAlchemy method.